### PR TITLE
added translation tasks

### PR DESCRIPTION
**Pull request by Vasilije1990**
## Summary by CodeRabbit

- **New Features**
	- Introduced a language detection feature that identifies the language of input text.
	- Added a translation function that uses AWS Translate to convert text between specified languages.

- **Bug Fixes**
	- Enhanced error handling and logging for both language detection and translation processes.

- **Chores**
	- Added a new dependency for improved language code handling.
	- Expanded configuration options for AWS integration by adding optional AWS credential fields.

### CODE REVIEW

File: cognee/modules/data/processing/detect_chunk_language.py'.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import boto3
```
Comment (borisarzentar) [inline cognee/modules/data/processing/detect_chunk_language.py'.py:1]: There is `'` in filename, probably mistake.

File: cognee/modules/data/processing/detect_chunk_language.py'.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        if detected_lang_iso639_1 == 'hr':
+            return 'sr'
+        return detected_lang_iso639_1
+
+    except LangDetectException as e:
+        logging.error(f"Language detection error: {e}")
+    except Exception as e:
+        logging.error(f"Unexpected error: {e}")
+
+    return -1
```
Comment (borisarzentar) [inline cognee/modules/data/processing/detect_chunk_language.py'.py:46]: Why return `-1`?

File: cognee/modules/data/processing/translate_chunks.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+from botocore.exceptions import BotoCoreError, ClientError
+from langdetect import detect, LangDetectException
+import iso639
+from dotenv import load_dotenv
+load_dotenv()
+
+import logging
+
+# Basic configuration of the logging system
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
```
Comment (borisarzentar) [inline cognee/modules/data/processing/translate_chunks.py:11]: We shouldn't configure the global logging object. Let's create a local logger and configure that one.

File: cognee/modules/data/processing/detect_chunk_language.py'.py (hunk 1 above)
Comment (borisarzentar) [reply cognee/modules/data/processing/detect_chunk_language.py'.py:1]: Do we want to detect it on the document level, or on the chunk level?

**Comment by borisarzentar** [comment]
@Vasilije1990 Please check comments

File: cognee/modules/data/processing/detect_chunk_language.py'.py (hunk 1 above)
Comment (Vasilije1990) [reply cognee/modules/data/processing/detect_chunk_language.py'.py:1]: How would tasks work on document level vs chunk from design side?

File: cognee/modules/data/processing/detect_chunk_language.py'.py (hunk 2 above)
Comment (Vasilije1990) [reply cognee/modules/data/processing/detect_chunk_language.py'.py:46]: No good reason

**Pull request closed without merging**